docs/unattend.xml

```diff
@@ -2,7 +2,7 @@
 <!--A sample unattend.xml which helps in setting admin password and running scripts on first boot-->
 <unattend xmlns="urn:schemas-microsoft-com:unattend">
    <settings pass="specialize">
-      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http:// www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-International-Core" processorArchitecture="am d64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http:// www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-International-Core" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
```

Comment:
there is also an extra space at ` xmlns:xsi="http:// www.w3.org/2001/XMLSchema-instance"` can you fix that as part of this commit.

Reply:
Good catch, will update. Thank you.

It's also present in the [documentation](https://github.com/openshift/openshift-docs/blob/master/modules/creating-the-vsphere-windows-vm-golden-image.adoc).

cc: @EricPonvelle